gensim/models/tfidfmodel.py

```diff
@@ -273,21 +277,34 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
                 * `c` - cosine.
 
             For more information visit [1]_.
-
+        pivot_norm : bool, optional
+            If pivot_norm is True, then pivoted document length normalization will be applied.
+        slope : float, optional
+            It is the parameter required by pivoted document length normalization which determines the slope to which
+            the `old normalization` can be tilted.
+        pivot : float, optional
+            Pivot is the point before which we consider a document to be short and after which the document is
```

Comment:
too "broad" description, next question will be "what is retrieval and relevence curves" and "how to plot it"